### DISCUSSION TITLE
OpenSSL 3.4.0 beta 1

### DISCUSSION THREAD

**Discussion by t8m** [openssl/openssl#25627] [Announcements]
Beta 1 of OpenSSL 3.4.0 is now available: please download and test it!

<hr /><em>This discussion was created from the release <a href='https://github.com/openssl/openssl/releases/tag/openssl-3.4.0-beta1'>OpenSSL 3.4.0 beta 1</a>.</em>

**Comment by Eimji** [comment]
Hello,

When do you plan to release the final 3.4 version?

Thanks.

**Reply by nhorman** [reply]
our development process currently has us creating a project planning board for each release, the timeline for which is visible here:
https://github.com/orgs/openssl/projects/12/views/5

On that plan, we intend to release on october 22
